lib/ansible/modules/packaging/os/redhat_repository.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/python
+
+# Giovanni Sciortino (giovannibattistasciortino@gmail.com)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: redhat_repository
+short_description: Manage RHSM repositories using the C(subscription-manager) command
+description:
+    - Manage(List/Enable/Disable) RHSM repositories to the Red Hat Subscription Management entitlement platform using the C(subscription-manager) command
```

Comment:
Standard YAML indentation is 2 spaces. The `EXAMPLES` block is using it so please change the indentation of the `DOCUMENTATION` block to 2 spaces as well.

Also please wrap all the lines in the `DOCUMENTATION` block on 80 characters. And please put period at the end of the description.